### ISSUE TITLE
please provide sapi5 version of espeak for windows.

### ISSUE THREAD

**Issue by zahra21** [espeak-ng/espeak-ng#1217]
hi.
i wish that you provide espeak sapi5 version for windows as you did before.
many people, need it like myself.
for example: i need espeak sapi5 that i can convert my favorite books and articles in to audio files.
many people maybe dont use nvda screen reader, they may just need text to speech capability,
or other people who use other screen readers.
windows xp, vista, seven, eight, ten, eleven are all support sapit5 and nvda itself supports sapi5 and even sapi4.
but i dont understand why espeak sapi5 version is not available anymore!
current espeak msi version is not suitable for novice users.
it has not any graphical interface in desktop, all programs, etc.
i installed it, but could not use it!
God bless you and His infinite mercy i pray for you!

**Comment by gregodejesus** [comment]
the sapi5 version is from espeak ng, but for some reason it does not run either in narrator, or in the sapi5 voices

**Comment by techsoulman** [comment]
I think this is what I need too. Is there any word on this?

**Comment by joshknnd1982** [comment]
I am still interested in an up-to-date sapi5 espeak ng. a good sapi5 espeak ng that works with 32 and 64bit sapi5 espeak still does not exist. if it did I could use espeak with audio games, balabolka, and windows11 narrator.

**Comment by joshknnd1982** [comment]
iphone has a small interface app so maybe sapi5 espeak should have a small interface app to let you control inflection word gap or pauses between words, volume rate and pitch of voice.

**Comment by joshknnd1982** [comment]
I am still interested in espeak ng sapi5 x64 and x86 so it works with both windows narrator and also older sapi5 programs. I submitted feedback to microsoft on this. I also need the ability to turn the inflection down to 20 or 30. and insert new variants because I like espeak ng glenn and others based on klatt6.

**Comment by joshknnd1982** [comment]
Would anyone else on here use espeak ng with narrator if it were available? I would. and I would also use it with audio games if I could use espeak Glenn with inflection set, to, 30% or 20%.

**Comment by 0x5066** [comment]
Constantly commenting about this on this issue isn't going to make it appear any faster.

**Comment by alex19EP** [comment]
indeed. please stop doing that. take this as the first official warning.

**Comment by NoahTCarver** [comment]
While I do understand the frustration with the repetitive comments above, I hope that they do not serve to downplay the usefulness of SAPI 5 compatibility. eSpeak NG is the preferred speech synthesizer of many screen reader users because of it's responsiveness and comprehensibility at high speeds, however it is at present only available to NVDA users because it is shipped with that screen reader. Many would prefer to use eSpeak NG with other Windows screen readers such as JAWS and Narrator. Additionally, the ability to use eSpeak NG with any SAPI 5- compatible program would provide easily-accessible text-to-speech functionality in languages for which speech engines do not yet exist. I therefore hope that developers might consider continuing development of SAPI 5 support as it's usefulness is readily apparent.

**Comment by gholamim6** [comment]
Hello guys.
I think it’d be useful to have this capability in binary builds of Espeak.
We can use Espeak-ng in SAPI compatible programs.
like Pot Player for reading the subtitles.
I know we can ask Pot Player to send Subtitles to screen reader directly, but reading them by SAPI TTS has its own benefits.
We can read Subs without interrupting screen readers speech.
None screen reader users also can use this capability.
I hope you would consider adding this useful feature in release binaries.
Thanks.

**Comment by willwade** [comment]
so as the readme says its in the original SF code  - if you are lucky (i.e. for languages supported) this works on Win 11. https://sourceforge.net/p/espeak/code/HEAD/tree/trunk/platforms/windows/windows_sapi/ - and installers https://master.dl.sourceforge.net/project/espeak/espeak/espeak-1.48/setup_espeak-1.48.02.exe?viasf=1

It was actually pretty neat. 

also : https://github.com/espeak-ng/espeak-ng/issues/966#issuecomment-860753965

and most importantly https://github.com/espeak-ng/espeak-ng/issues/7 - ie its being slowly worked on
